### PR TITLE
Add FAQ/misc: How to use the official installation container with docker/podman

### PR DESCRIPTION
**Pull request by Reamer**
The following workaround was very helpful to solve the missing libvirt libraries under Ubuntu.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -169,3 +170,36 @@ This one is very helpful if you want to know if a certain commit has landed in y
     ...
 
   ```
+## How can I install OKD without libvirt libraries?
```
Comment (vrutkovs) [inline FAQ.md:173]: The plan is to have a dedicated libvirt installer in 4.8 as it really annoys people (sorry :( ).

Lets make it more generic - official installer container is what we use in CI, so this should be renamed to "Official installer container" and focus less on libvirt libs

Comment (Reamer) [reply FAQ.md:173]: Okay

Comment (Reamer) [reply FAQ.md]: I have changed that part. Please check again.

File: FAQ.md [hunk 2]
```diff
@@ -16,13 +16,14 @@ Below are answers to common questions regarding OKD installation and administrat
 # Misc #
 - [Interesting commands while an upgrade runs](#interesting-commands-while-an-upgrade-runs)
 - [How can I find out what's inside of a (CI) release and which commit id each component has?](#how-can-i-find-out-whats-inside-of-a-ci-release-and-which-commit-id-each-component-has)
+- [How can I install OKD without libvirt libraries?](#how-can-i-install-okd-without-libvirt-libraries)
```
Comment (LorbusChris) [inline FAQ.md:19]: this doesn't match the newly added heading

**Review by LorbusChris — CHANGES_REQUESTED** [review]

File: FAQ.md (hunk 2 above)
Comment (Reamer) [reply FAQ.md]: Thank you very much for your review. I have corrected this.

**Comment by LorbusChris** [comment]
Thanks @Reamer!

**Pull request merged**